src/SmartComponents/Actions/ViewActions.js

```diff
@@ -138,11 +140,21 @@ class ViewActions extends Component {
         return parsedTitle.length > 1 ? `${parsedTitle[0]} ${parsedTitle[1]} Actions` : `${parsedTitle}`;
     }
 
+    onNavigate(_event, _item, key) {
```

Comment:
also this probably should be abstracted into the same helper, imported into the file that uses it, as code climate would say... duplicate code found 😩